worlds/alttp/docs/multiworld_en.md

```diff
@@ -6,10 +6,20 @@
 `SNI Client - A Link to the Past Patch Setup`
 - [SNI](https://github.com/alttpo/sni/releases). This is automatically included with your Archipelago installation above.
 - SNI is not compatible with (Q)Usb2Snes.
-- Hardware or software capable of loading and playing SNES ROM files
+- Hardware or software capable of loading and playing SNES ROM files, including:
     - An emulator capable of connecting to SNI
       ([snes9x rr](https://github.com/gocha/snes9x-rr/releases),
        [BizHawk](https://tasvideos.org/BizHawk), or
+      ([snes9x-nwa](https://github.com/Skarsnik/snes9x-emunwa/releases), [snes9x rr](https://github.com/gocha/snes9x-rr/releases),
+       [BSNES-plus](https://github.com/black-sliver/bsnes-plus),
+       [BizHawk](http://tasvideos.org/BizHawk.html), or
+       [RetroArch](https://retroarch.com?page=platforms) 1.10.1 or newer)
+    - An SD2SNES, [FXPak Pro](https://krikzz.com/store/home/54-fxpak-pro.html), or other compatible hardware. **note: 
+modded SNES minis are currently not supported by SNI. Some users have claimed success with QUsb2Snes for this system,
+but it is not supported.**
+    - An emulator capable of connecting to SNI
+      ([snes9x-nwa](https://github.com/Skarsnik/snes9x-emunwa/releases), [snes9x rr](https://github.com/gocha/snes9x-rr/releases),
+       [BizHawk](http://tasvideos.org/BizHawk.html), or
        [RetroArch](https://retroarch.com?page=platforms) 1.10.1 or newer). Or,
```

Comment:
hm, that section got duplicated. I think the suggestion was not meant to be applied as-is. :/
Once that's fixed I'm gonna merge it and we can revert if people find bugs.

Reply:
Done. My apologies for missing this. I also removed the duplicate mentions of snes9x-rr and added the missing dash for it.